3_javascript/javascript1/week1/JS-Smart-ease/Housey-pricey/house-price.js

```diff
@@ -0,0 +1,31 @@
+// Peter
+let widthInM2= 8;
+let depthInM2= 10;
+let heightInM2= 10;
+let gardenSizeInM2 = 100;
+const house1= 2500000;
+
+volumeInMeters = widthInM2 * depthInM2 * heightInM2; 
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+console.log(housePrice);
+
+if ((house1-housePrice)>0){
+    console.log("The house price for Peter is expensive, and it should be no more than "+ housePrice);
+} else {
+    console.log("The house price for Peter is cheap.")
+};
+
+// Julia
+
+const house2= 1000000;
```

Comment:
I would suggest escaping generic variable names and instead choosing names that tell what the variable is used for. That will make the code more readable. So instead of house1 and house2, I would suggest to use thePricOfTheHouseJuliaWants and thePriceOfTheHousePeterWants.